src/Symfony/Contracts/Service/Attribute/Required.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Contracts\Service\Attribute;
+
+use Attribute;
+
+/**
+ * Flags a public setter, wither or property for autowiring.
```

Comment:
Wrong :)
The difference is stubble yet critical: this is *not* for autowiring.
This is to improve the expressivity of the source code, to tell that this property or method should be considered as part of the instantiation steps of this class.
Then, it happens that this info is useful to autowiring.
But let's not short-circuit the meaning of things please :)

Reply:
You are right! I'll try to find a better description.

Reply:
Please have a look if the new description is a better fit.